src/include/maat/cpu.hpp

```diff
@@ -76,9 +76,9 @@ class CPUContext: public serial::Serializable
 
 private:
     // Internal method that handles setting register aliases
-    inline void _set_aliased_reg(ir::reg_t reg, const Value& val) __attribute__((always_inline));
+    inline ALWAYS_INLINE void _set_aliased_reg(ir::reg_t reg, const Value& val);
     // Internal method to check if a register is an alias
-    inline bool _is_alias(ir::reg_t reg) const __attribute__((always_inline));
+    inline ALWAYS_INLINE bool _is_alias(ir::reg_t reg) const;
```

Comment:
In general most of these uses of `inline` like this are meaningless (it's just for symbol visibility and class methods are `inline` per default). I kept it this way to keep the changes minimal.